Add Incident Response to billing docs (#32244)
DataDog/documentation: content/en/account_management/billing/incident_response.md

Organizations have two options for incident response billing:
- Incident Management or On-Call as separate SKUs, each billed individually
- The Incident Response bundle, which unifies billing for both Incident Management and On-Call under a single SKU

Each product bills by seat, so each user who performs qualifying actions requires a seat license. For detailed pricing information, see the [Incident Response pricing page][3].

View and manage seat usage for each product in the [Seat Management][4] section of the Plan & Usage page in Datadog.

## Allocate seats

A **Seat** is a license for a user to actively participate in the on-call or incident management process in Datadog. 

Any user can claim a seat for On-Call, Incident Management, or Incident Response as they perform actions that require a seat. Billing admins can also assign seats ahead of time to simplify the process for their teams.

A user needs an **On-Call seat** if they perform any of the following:
- Participate in an [on-call schedule][5]
- Are included in an [escalation policy][6]
- Have set up [notification preferences][7] to receive pages

> In the On-Call product, anyone can view existing pages, escalation policies, teams, and schedules without a seat. 

Reviewer: ```suggestion
In the On-Call product, a user does not need a seat to view existing pages, escalation policies, teams, or schedules.
```

I think this ordering makes the clausal relationships a little clearer
